dashboard/app/views/devise/registrations/edit.html.haml

```diff
@@ -79,6 +83,9 @@
       - location = Geocoder.search(request.ip).try(:first)
       - country_code = location&.country_code.to_s.upcase
       = f.label :us_state, t('activerecord.attributes.user.us_state'), class: 'label-bold'
+      - if studentInLockoutFlow
+        %em= t('user.update_field_parent_permission_required')
+        %br/
       = f.select :us_state, us_state_options, {include_blank: t('school_info.select_state')}, required: true
```

Comment:
if we do this (and for age as well) do we get to not pass things to JavaScript to disable?

```suggestion
      = f.select :us_state, us_state_options, {include_blank: t('school_info.select_state')}, required: true, disabled: !!Policies::ChildAccount.lockout_date(current_user)
```